feature/details/src/main/java/at/marki/moviedb/feature/details/DetailsViewModel.kt

```diff
@@ -45,6 +45,9 @@ class DetailsViewModel @Inject constructor(
                     )
                 }
             }
+        //REVIEW: the debounce here actually makes it a bit worse,
+        // as the bottomsheet will still show the data it was showing before
+        // during the debounce time and skip the loading state
```

Comment:
Yeah, I modified this to be set to Loading on closing the bottom sheet. Then the debounce doesn't really interfere (but is still unnecessary here)